### DISCUSSION TITLE
Github API does not return latest build

### DISCUSSION THREAD

**Discussion by Vunzz** [microsoft/winget-cli#1511] [Q&A]
Hi,

I am busy creating a little script, to always download the latest release from Github, using the Github API: https://api.github.com/repos/microsoft/winget-cli/releases/latest.

Somehow the API refers to the previous version: https://github.com/microsoft/winget-cli/releases/download/v1.0.11692/Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle instead of  https://github.com/microsoft/winget-cli/releases/download/v1.0.12576/Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle.

Powershell:
`Invoke-WebRequest -Uri (Invoke-WebRequest -Uri "https://api.github.com/repos/microsoft/winget-cli/releases/latest" | ConvertFrom-Json).Assets.browser_download_url[2] -OutFile ".\Downloads\wingetupdate.msixbundle"`

**Comment by ItzLevvie** [comment]


**Reply by Vunzz** [reply]
I wish everything was so simple 😉
Although the pre-release tag is easy to overlook.
![I learned something today](https://i0.wp.com/i.redd.it/so7077ci70o31.png?w=750&ssl=1)

**Reply by Vunzz** [reply]
Bonus:

Powershell
`Invoke-WebRequest -Uri ((Invoke-WebRequest -Uri "https://api.github.com/repos/microsoft/winget-cli/releases" | ConvertFrom-Json).Assets.browser_download_url | Where-Object -FilterScript {$_ -like "*.msixbundle"} | Sort-Object | Select-Object -Last 1) -OutFile .\Downloads\wingetupdate.msixbundle && Add-AppPackage -Path .\Downloads\wingetupdate.msixbundle`
